CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Change Log
 
+## [1.4.5] - 2023-04-04
+
+- **Chore** - CI pipeline updated with GO 1.20.4
```

Comment:
As a user I don't know what that means. maybe something like `Chore - Backend binaries are now compiled with golang 1.20.4`